main.go

```diff
@@ -150,7 +142,7 @@ func run() error {
 		return fmt.Errorf("the environment variable BOT_SIGNING_SECRET must be set")
 	}
 
-	prowJobKubeconfig, _, _, err := loadKubeconfig()
+	prowJobKubeconfig, _, _, err := manager.LoadKubeconfig()
```

Comment:
While I understand how these changes came about, I dont think the methods related to loading of kubeconfigs should be part of the `manager` package.  They are really only used in `main` and really just muddy up the manager code.

Reply:
Moved the method to `main.go`